packages/bot-web-ui/src/app/app.tsx

```diff
@@ -12,6 +12,10 @@ type TAppProps = {
 };
 
 const App = ({ passthrough }: TAppProps) => {
+    // Setting the inner height of the document to the --vh variable to fix the issue of dynamic view height(vh) on mobile browsers for few scrollable components
+    const vh = window.innerHeight;
+    document.body.style.setProperty('--vh', `${vh}px`);
+
```

Comment:
Can this be handled in css itself? If not can we make it a utility and just call it here?

Reply:
From CSS itself, we were using vh which is actually the whole screen for mobile devices that includes the browser and top bar. Which is causing the scrollbar issue. The other workarounds are dvh, svh and lvh (a bit performance-heavy according to this reference https://ishadeed.com/article/new-viewport-units/#:~:text=The%20problem,Safari%20or%20Chrome%20on%20Android). This is working fine initially but upon opening the accordion the height calculation is not working accordingly and scrolling into view is shifting the whole div to the top. That's why created this utility so that we can use it for this type of case. 

Keeping it to the utility should be the ideal way. Let me move it.

Reply:
Done ✅